src/components/main-dashboard/WelcomeHeader.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from 'react';
+import {
+  getCurrentDate,
+  getCurrentTime,
+} from '../../utilities/helperFunctions';
+
+export default function WelcomeHeader() {
+  const [time, setTime] = useState(getCurrentTime());
+  const currentDay = getCurrentDate();
```

Comment:
This is an edge case which could be separated into another project issue, but it looks like the way this is rendered now, the day won't update if the user is viewing the dashboard through midnight.   

Thinking about this issue, it similarly applies to other components like the hourly forecast which would also need to update if an hour passed while the user was viewing the dashboard.

Reply:
Yeah I realized that this doesn't continously update the day if they happen to view it at midnight. I was thinking about this as well as the hourly weather and assumed the user could just refresh the browser for an update but that isn't very effective. I can look into changed this!